docs/decorator-reference.md

```diff
@@ -99,7 +99,7 @@ export class User {
 
 `@Column` accept several options you can use:
 
-* `type: ColumnType` - Column type. One of the type listed [above](#column-types).
+* `type: ColumnType` - Column type. One of the [supported column types](./entities.md#column-types).
```

Comment:
i guess website will not create a proper non-github link...

Reply:
What do you mean with "non-github link"?